c2corg_ui/static/js/alertservice.js

```diff
@@ -0,0 +1,54 @@
+goog.provide('app.Alerts');
+
+
+goog.require('app');
+
+
+
+/**
+ * @constructor
+ * @export
+ */
+app.Alerts = function() {
+
+  /**
+   * @type {Array.<appx.AlertMessage>}
+   * @private
+   */
+  this.alerts_ = [];
+};
+
+
+/**
+ * @param {string|appx.AlertMessage} msg Alert data.
```

Comment:
Instead of mixing the types, I would make `type` and `timeout` optional arguments. So that you could call the method with:

    this.alerts_.add(msg);

or with:


    this.alerts_.add(msg, 'danger', 5000);